src/Jackett.Common/Plumbing/JackettModule.cs

```diff
@@ -45,19 +43,14 @@ protected override void Load(ContainerBuilder builder)
             switch (_runtimeSettings.ClientOverride)
             {
                 case "httpclientnetcore":
-                    RegisterWebClient<HttpWebClientNetCore>(builder);
-                    break;
-                case "httpclient2netcore":
-                    RegisterWebClient<HttpWebClient2NetCore>(builder);
-                    break;
                 case "httpclient":
                     RegisterWebClient<HttpWebClient>(builder);
                     break;
+                case "httpclient2netcore":
```

Comment:
The code is good but we can let a warning message with something like "the client httpclient2netcore was removed, using httpclient2 instead". Maybe some users are using ClientOverride parameter.

Reply:
I'm going to be collapsing httpClient and httpClient2 in a future PR as well. At that point we can add a warning that the entire ClientOverride parameter is going to be removed as it doesn't do anything.

Reply:
ok